src/client/App.js

```diff
@@ -1,12 +1,16 @@
 import React from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import TestComponent from "./components/TestComponent/TestComponent";
+import MealListComponent from "./components/MealListComponent";
 
 function App() {
   return (
     <Router>
+      <Route exact path="/all-meals">
+        <MealListComponent />
+      </Route>
       <Route exact path="/">
-        <p>test</p>
+        <p>testa</p>
```

Comment:
👀

Reply:
This is how I tested frontend before writing main code 🤓